ui/src/components/forms/UserFormContainer.js

```diff
@@ -52,13 +52,14 @@ const UserFormContainer = (props) => {
       {!isLoading && !allowFormAccess && (
         <Center h="100vh">
           <VStack>
-            <Text fontSize={["xl", "3xl"]} spacing="5">
-              Uh-oh!
+            <Text fontSize={['xl', '3xl']} spacing="5">
+              You are unable to access this form. Please see the message below
+              for instructions
             </Text>
-            <Text fontSize={["lg", "2xl"]} spacing="5">
+            <Text fontSize={['lg', '2xl']} spacing="5">
               {user.id && !user.hasFilledProfileForm
-                ? "Please complete your user profile first before signing up"
-                : "You need to be logged in to access this form or this form is currently unavailable!"}
+                ? 'Please complete your user profile first before signing up'
```

Comment:
maybe Feb2022R1 we should make a bunch of different error pages

Reply:
agree